src/main/java/com/ithillel/service/appcontext/PropertiesApplicationContext.java

```diff
@@ -0,0 +1,50 @@
+package com.ithillel.service.appcontext;
+
+import com.ithillel.service.storage.Storage;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+public class PropertiesApplicationContext implements ApplicationContext {
+
+    private static final String TEXT_PROCESSOR = "textProcessor";
+    private Map<String, Object> beans = new HashMap<>();
+
+    public PropertiesApplicationContext() {
+        Properties applicationProperties = new Properties();
+        try {
+            applicationProperties.load(getClass().getClassLoader()
```

Comment:
if you not load applicationProperties there don't have sense to move to the next steps, please provide correct behavior
- rm try - catch in the middle of code(can replace with "global" try catch for whole method)
- e.printStackTrace(); yes, -  it;s good for debug, but not for production, can use logger instead, or as alternative sout(tmp solution with https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/exception/ExceptionUtils.html))